CONVENTIONS.md

```diff
@@ -0,0 +1,75 @@
+---
+title: conventions
+authors:
+  - "@smarterclayton"
+  - "@derekwaynecarr"
+  - "@jwforres"
+  - "@crawford"
+  - "@eparis"
+  - "@russellb"
+  - "@markmc"
+reviewers:
+  - "@smarterclayton"
+  - "@derekwaynecarr"
+  - "@jwforres"
+  - "@crawford"
+  - "@eparis"
+  - "@russellb"
+  - "@markmc"
+approvers:
+  - "@smarterclayton"
+  - "@derekwaynecarr"
+  - "@jwforres"
+  - "@crawford"
+  - "@eparis"
+  - "@russellb"
+  - "@markmc"
+creation-date: 2020-05-19
+last-updated: 2020-05-19
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# OpenShift Conventions
+
+## Summary
+
+This document identifies conventions adopted by the OpenShift project
+as a whole. Generally the conventions outlined below should be followed by all
+components that make up the project, but exceptions are allowed when necessary.
+
+
+## Motivation
+
+The conventions are intended to help with consistency across the project. Users
+of the platform expect this consistency in the experience and operation of the cluster.
```

Comment:
Developers of the platform expect this consistency in the code to quickly identify issues across codebases.

Maybe a comment too about how consistency simplifies explanation, reduces accidental complexity.